libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -2296,24 +2424,160 @@ def version_header(self) -> str:
             )
         )
 
+    def header_ftm(self, header: str) -> Dict[Std, List[Dict[Ftm, FtmHeaderTest]]]:
```

Comment:
I think the name of this method is somewhat confusing, since this is really generating test-related data only.

Reply:
I changed the name, but I had a bit of an issue finding a good name.